cli/wallet/wallet.go

```diff
@@ -160,6 +164,20 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:      "remove-account",
```

Comment:
Maybe just `remove`? What else do we have in the wallet?

Reply:
I thought about `remove`, but wanted to mirror `create-account`.

Reply:
Maybe we can rename `wallet create` to something else then.

Reply:
`wallet create` -> `wallet initialize` and then strip `-account` everywhere.